app/src/main/resources/db/Rows.sql

```diff
@@ -1,95 +1,95 @@
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Great Lakes Leviathan','levi.jpeg',1,1,'critter',2,4,NULL,'Rising from the deep with monstrous might, Great Lakes Leviathan crashes through waters, dragging foes down into its cold, murky depths!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Maple Syrup Monster','syrup.jpeg',4,1,'critter',2,4,NULL,'Sticky, sweet, and absolutely terrifying, Maple Syrup Monster oozes through the battlefield, trapping foes in a sugary mess they can’t escape!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Rocky Mountain Raptor','raptor.jpeg',3,2,'critter',2,4,NULL,'Soaring through the peaks with deadly precision, Rocky Mountain Raptor swoops down, striking hard and leaving its mark on the competition!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Timberwolf Alpha','wolf.jpeg',3,2,'critter',1,3,NULL,'Leading the pack with fierce loyalty and razor-sharp claws, Timberwolf Alpha dominates the battlefield, hunting down foes with savage precision!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Ontario Ogre','ogre.jpeg',2,2,'critter',3,4,NULL,'Lumbering through the land with a gruff growl, Ontario Ogre smashes foes with boulders and brute strength, leaving a trail of wreckage and crushed pride!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Manitoba Mauler','manit.jpeg',1,3,'critter',2,3,NULL,'Charging through the snow with bone-crushing force, Manitoba Mauler tears through enemies, leaving nothing but destruction and frozen regrets in its wake!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Enraged Prime Minister','prime.jpeg',3,4,'critter',4,5,NULL,'Raging with unchecked power, Enraged Prime Minister storms the battlefield, making bold moves and crushing opposition with policies of pure chaos!');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('Rob (Base Form)','normal.png',2,0,'critter',1,3,NULL,'Your internet friend.');
+INSERT INTO CARDS (name, image, playCost, rarity, type, damage, health, effectid, description) VALUES ('naireduG boR','reverse.png',2,0,'critter',3,1,NULL,'NaireduG boR storms in, armed with backwards logic and a grin that suggests he’s about to turn your world inside out—expect the unexpected and nothing at all to make sense!');
```

Comment:
this should be the normal rob description but backwards lol

Reply:
done lol